### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 26

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#202] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254277-1cfe5231-ac7c-4e14-b1ad-b951b1f75586.png)

**Comment by Exyss** [comment]
- 10 lettere totali
- 6 lettere N
- 3 lettere A
- 1 lettere I

**Risposte:**
1. $\dfrac{10!}{6! \cdot 3! \cdot 1!}$

2. Risolto per doppio conteggio:
        - Per calcolo diretto escludendo le A per la prima posizione:
  $\dfrac{7 \cdot 9!}{6! \cdot 3! \cdot 1!}$ 
          - Per complemento tra il totale e i casi con solo A in prima posizione:
  $\dfrac{10!}{6! \cdot 3! \cdot 1!} - \dfrac{3 \cdot 9!}{6! \cdot 3! \cdot 1!} = \dfrac{9!(10-3)}{6! \cdot 3! \cdot 1!} = \dfrac{7 \cdot 9!}{6! \cdot 3! \cdot 1!}$
  
3. $(InizianoNN \cup FinisconoNN) = \dfrac{6 \cdot 5 \cdot 8!}{6! \cdot 3! \cdot 1!} + \dfrac{8! \cdot 6 \cdot 5}{6! \cdot 3! \cdot 1!} - \dfrac{6 \cdot 5 \cdot 6! \cdot 4 \cdot 3}{6! \cdot 3! \cdot 1!}$

**Comment by matypist** [comment]
2. Triplo conteggio (giusto per aggiungerne un altro):

- #{casi che iniziano per I} + #{casi che iniziano per N} = 9!/(6! * 3!) + 9!/(5! * 3!) [sempre 588]

3. Stesso metodo (PIE) e stesso risultato di @Exyss, solo scritto diversamente:
- #A = #{Iniziano con NN} = (8!)/(4!3!)
- #B = #{Finiscono con NN} = (8!)/(4!3!)
- #{A ∩ B} = #{Iniziano E finiscono con NN} = (6!)/(2!3!)
- #{A ∪ B} = #{Iniziano O finiscono con NN} = #A + #B - #{A ∩ B}
